application/src/main/java/org/togetherjava/tjbot/features/moderation/modmail/ModMailCommand.java

```diff
@@ -78,7 +82,7 @@ public ModMailCommand(JDA jda, Config config) {
 
         guildOption.addChoices(choices);
 
-        getData().addOptions(messageOption, guildOption, anonymousOption);
+        getData().addOptions(/* messageOption, */ guildOption, anonymousOption);
```

Comment:
this commented out section looks like a debugging left-over

Reply:
Ops!